src/pushsource/_impl/model/ami.py

```diff
@@ -99,9 +99,42 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Notes regarding changes in the newest version. URLs are supported.
```

Comment:
What does it mean for URLs to be "supported"?  If it's just a string, couldn't you put anything in it?

Reply:
I did included the 'URL' part as it was mentioned in AWS info for release_notes, I think it means if they found any url they will be highlighting that, but in this case too, I think explicitly mentioning this one does not make it any better, so I will remove it.

Reply:
fixed this one.